### ISSUE TITLE
Gauss-Seidel max iterations and convergence

### ISSUE THREAD

**Issue by mshoe** [dilevin/CSC2549-a6-rigid-body-contact#4]
When I use 10 PGS iterations, the simulation is very fast when the torus is not contacting the floor, and significantly slower when it is. This gives a very temporally jumpy looking simulation.

But when I use 1 PGS iteration, it seems to run fairly smoothly and the contact is still resolved just like in the readme gif.

Is this jumpy behavior with 10 iterations expected? I believe so since PGS should be the bottleneck of the simulation. So should we submit it with 10 iterations or 1 iteration which also works well.

**Comment by dilevin** [comment]
yes, pgs is the bottleneck and only runs when the objects in contact with the floor, so the more pigs iterations you use, the more time it takes to compute a time step during contact resolution
